dd-sdk-android/src/main/kotlin/com/datadog/android/rum/ActivityLifecycleTrackingStrategy.kt

```diff
@@ -65,4 +65,28 @@ abstract class ActivityLifecycleTrackingStrategy : Application.ActivityLifecycle
     }
 
     // endregion
+
+    // region Utils
+
+    /**
+     * Maps the Bundle key - value properties into compatible attributes for the Rum Events.
+     * @param bundle the Bundle we need to transform. Returns an empty Map if this is null.
+     */
+    protected fun asRumAttributes(bundle: Bundle?): Map<String, Any?> {
```

Comment:
the name `asRumAttributes` works for an extension function, but a method should eb named with a verb, e.g.: `extractRumAttributes` or `convertToRumAttributes`